app/frontend/src/utils/transformUtils.js

```diff
@@ -42,3 +43,24 @@ export function parseIdps(identityProviders) {
   }
   return result;
 }
+
+/**
+ * @function attachAttributesToLinks
+ * Attaches attributes to <a> Link tags to open in a new tab
+ * @param {Component[]} formSchemaComponents An array of Components
```

Comment:
IIRC the JSDoc doesn't support custom types like Component unless you define it somewhere else what that type is. I'd recommend changing this to just be `Object[]` and let the description handle describing what kind of object array it is.